inc/configuration/class-yoast-acf-analysis-configuration.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+
+/**
+ * Class Yoast_ACF_Analysis_Configuration_Default
+ */
+class Yoast_ACF_Analysis_Configuration {
+
+	/**
+	 * @var Yoast_ACF_Analysis_String_Store
+	 */
+	protected $blacklist;
+
+	/**
+	 * @var Yoast_ACF_Analysis_String_Store
+	 */
+	protected $field_selectors;
+
+	/**
+	 * @param Yoast_ACF_Analysis_String_Store  $blacklist       Blacklist Configuration Object.
+	 * @param Yoast_ACF_Analysis_String_Store $field_selectors Field Selectors Configuration Object.
+	 */
+	public function __construct( Yoast_ACF_Analysis_String_Store $blacklist, Yoast_ACF_Analysis_String_Store $field_selectors ) {
+		$this->blacklist       = $blacklist;
+		$this->field_selectors = $field_selectors;
+	}
+
+	/**
+	 * @return string
+	 */
+	public function acf_version() {
+		return get_option( 'acf_version' );
+	}
+
+	/**
+	 * @return Yoast_ACF_Analysis_String_Store
+	 */
+	public function blacklist() {
+
+		$blacklist = apply_filters(
+			Yoast_ACF_Analysis_Facade::get_filter_name( 'blacklist' ),
```

Comment:
Just out of curiosity interested in why you've chosen to use strings here instead of `__FUNCTION__` like I've been using in the earlier implementation.